Public/Src/Utilities/UnitTests/Executables/TestProcess/Operation.cs

```diff
@@ -853,10 +876,10 @@ private void DoEnumerateDir()
 
         private void DoCreateSymlink()
         {
-            string target = AdditionalArgs ?? FileOrDirectoryToString(Path);
+            string target = AdditionalArgs ?? PathAsString;
 
             // delete whatever might exist at the link location
-            var linkPath = FileOrDirectoryToString(LinkPath);
+            var linkPath = LinkPathAsString;
             FileUtilities.DeleteFile(linkPath);

```

Comment:
```suggestion
            if (FileUtilities.Exists(linkPath))
            {
                FileUtilities.DeleteFile(linkPath);
            }
```

Reply:
FileUtilities.DeleteFile() doesn't fail if the file isn't there, so making that change yields no behavioral difference.

Reply:
The assert failing in the unit tests is due to link path being empty/null I thought. `Exists` should return false on invalid path and thus we don't fail the assert in `DeleteFile`. I thought that was macOS unit test failure I saw.

Reply:
Oh, I missed your point. Previously the FileOrDirectoryToString(LinkPath) would have been returning a string with "{Invalid}" if the LinkPath weren't specified. So it would have been calling FileUtilities.DeleteFile() with a path that didn't reference a real file. Now it's calling it with an empty string. I admittedly hadn't seen the test failure you were talking about since the pipeline is otherwise broken

Reply:
Switched it to default to {Invalid} for the filename to preserve the prior implementation's behavior